misc-scripts/distribution.py

```diff
@@ -0,0 +1,140 @@
+import os
+import argparse
+import requests as req
+import json
+import subprocess
+import time
+
+from dotenv import load_dotenv
+
+load_dotenv()
+
+parser = argparse.ArgumentParser()
+
+def node_type(x):
+    x = int(x)
+    if x < 2:
+        raise argparse.ArgumentTypeError(f"The number of nodes should be Min. 2, you have entered {x}")
+    return x
+
+parser.add_argument('nodes', type= node_type, help= 'Number of nodes should be Min. 2 should be given')
+
+args = parser.parse_args()
+print("**** Number of nodes mentioned: %d *****" % args.nodes)
+
+url = 'https://ipinfo.io/json'
+resp = req.get(url)
+l = json.loads(resp.text)
+IP = l["ip"]
+
+if not len(IP):
+    IP='127.0.0.1'
+
+print("---------- Run withdraw rewards tx -------")
+
+DAEMON = os.getenv('DAEMON')
+DAEMON_HOME= os.getenv('DAEMON_HOME')
+CHAINID = os.getenv('CHAINID')
+DENOM = os.getenv('DENOM')
+for a in range(1,args.nodes+1):
+    DIFF = a-1
+    INC = DIFF*2
+    PORT = 16657 + INC
+    RPC = f"http://{IP}:{PORT}"
+    validator = f"{DAEMON} keys show validator{a} --bech val --keyring-backend test --home {DAEMON_HOME}-{a} --output json"
+    process = subprocess.Popen(validator.split(), stdout=subprocess.PIPE, text=True)
+    output, error = process.communicate()
+    res = (json.loads(output))
+    VALADDRESS = res["address"]
+    FROMKEY = "validator"+str(a)
+    print(f"** validator address :: {VALADDRESS} and From key :: {FROMKEY} **")
+    print(f"Iteration no {a} and values of address : {VALADDRESS} and key : {FROMKEY}")
+    print(f"--------- withdraw-rewards of {FROMKEY} -----------")
```

Comment:
From lines 44 to 52: Don't have the print the error use `subprocess.run(['command1', 'command2'])` It will automatically prints the error.